common/context/src/main/java/io/helidon/common/context/Context.java

```diff
@@ -162,6 +163,20 @@ static Builder builder() {
      */
     String id();
 
+    /**
+     * Unmodifiable collection of all classifiers registered in this context.
+     *
+     * @return collection of classifiers
+     */
+    Collection<Object> classifiers();
```

Comment:
The context is not considering classifiers to be property names - there is no semantic restriction.
Allowing the list of classifiers to escape is actually not desired - as these may be package private classes.

If you want to use `Context` as a way to propagate Jersey properties across threads, please use a custom object.

```
context.register(MyClass.class, new Properties());
then you can call add/list/remove on the properties object, rather then on context.
```

Reply:
OK, that sounds reasonable.